### ISSUE TITLE
Set up travis

### ISSUE THREAD

**Issue by lawnsea** [TreehouseJS/broker#11]
- [ ] set up a TreehouseJS account on Travis
- [ ] configure travis to run tests in phantom (depends on #9)
- [ ] configure travis to run tests on saucelabs (depends on #10)

We definitely want to run the tests on Sauce Labs. However, if #10 takes a while
to fix, let's just run the tests on phantom at first and file a separate issue
for the last task.
